packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -91,6 +125,10 @@ export const PaymentUpdateForm = ({
   const onPayment = useCallback(
     (tokenResponse: stripe.TokenResponse) => {
       if (tokenResponse && tokenResponse.token) {
+        if (useSCAPaymentFlow) {
+          // Use SetupIntents to Confirm the Card
+          handleSCA(tokenResponse.token.id);
```

Comment:
FWIW, I know this is in progress, but wanted to mention: 

We're also going to want to swap in [`PaymentFormV2`](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/src/components/PaymentFormV2/index.tsx) over here when `useSCAPaymentFlow` is true. 

That uses the new Stripe elements library. It also has [a different `onSubmit` callback](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.tsx#L87) that replaces `onPayment` / `onPaymentError` callbacks no longer generates a payment token.